src/ui/public/field_editor/field_editor.html

```diff
@@ -94,7 +110,48 @@ <h4 class="hintbox-heading">
     </div>
 
     <div class="form-group">
-      <div ng-bind-html="editor.scriptingInfo" class="hintbox"></div>
+      <div class="hintbox">
+        <h4>
+          <i class="fa fa-question-circle text-info"></i> Scripting Help
+        </h4>
+
+        <p>
+          By default, Kibana scripted fields use <a target="_window" ng-href="{{editor.docLinks.painless}}">Painless <i class="fa-link fa"></i></a>, a simple and secure scripting language designed specifically for use with Elasticsearch.  To access values in the document use the following format:
```

Comment:
Yes, but if you merge this change before we have a 5.0 branch it will break the docs build.